assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -53,276 +65,517 @@ export const PRE_PUBLISH_MESSAGE_TYPES = {
+            {__(
+              'Use a page attachment breaking up the text into multiple screens; or',
+              'web-stories'
+            )}
+          </li>
+          <li>
+            {sprintf(
+              /* translators: %d: maximum percentage of characters, depending on number of pages. */
+              __(
+                'Keep the total number of text-heavy pages under %d of total number of story pages',
+                'web-stories'
+              ),
+              MAX_CHARACTER_PERCENTAGE + '%'
+            )}
+          </li>
+        </DashedList>
       ),
     },
     TOO_LITTLE_TEXT: {
-      MAIN_TEXT: __('Too little text', 'web-stories'),
-      HELPER_TEXT: sprintf(
-        /* translators: %d: minimum number of story characters. */
-        __(
-          'The entire story should have a minimum of %d characters of text in total.',
-          'web-stories'
-        ),
-        MIN_STORY_CHARACTERS
+      MAIN_TEXT: __('Increase amount of total text', 'web-stories'),
+      HELPER_TEXT: (
+        <DashedList>
+          <li>
+            {sprintf(
+              /* translators: %d: minimum number of story characters. */
+              _n(
+                'Make your story text %d character or more',
+                'Make your story text %d characters or more',
+                MIN_STORY_CHARACTERS,
+                'web-stories'
+              ),
+              MIN_STORY_CHARACTERS
+            )}
+          </li>
+        </DashedList>
       ),
     },
   },
   MEDIA: {
     VIDEO_IMAGE_TOO_SMALL_ON_PAGE: {
-      MAIN_TEXT: __('Video or image too small on the page', 'web-stories'),
+      MAIN_TEXT: __('Increase video or image size on the page', 'web-stories'),
       HELPER_TEXT: (
-        <TranslateWithMarkup
-          mapping={{
-            a: (
-              //eslint-disable-next-line jsx-a11y/anchor-has-content
-              <a
-                href="https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/#visual-treat"
-                rel="noreferrer"
-                target="_blank"
-              />
-            ),
-          }}
-        >
-          {__(
-            'Use full screen videos and images where possible to keep to an immersive feeling. <a>(more info)</a>',
-            'web-stories'
-          )}
-        </TranslateWithMarkup>
+        <DashedList>
+          <li>
+            <TranslateWithMarkup
+              mapping={{
+                a: (
+                  //eslint-disable-next-line jsx-a11y/anchor-has-content
+                  <a
+                    href="https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/#visual-treat"
+                    rel="noreferrer"
+                    target="_blank"
+                  />
+                ),
+              }}
+            >
+              {__(
+                '<a>Use full screen videos and images whenever possible to create a more immersive experience. (more info)</a>',
```

Comment:
nit: this is the only piece of text with a period at the end of the list item. Thoughts on removing it for consistency?